### PR TITLE
Dockerfile setup

### PR DESCRIPTION
**Pull request by andrew21-mch**
this solves issue #26 
Created Dockfile to setup configurations to run docker image of miniyotas
updated the readme with instructions on how to build docker image

please could you checkout and confirm if this is what is need, thanks

### CODE REVIEW

File: src/res/osscameroon_issues.json [hunk 1]
```diff
@@ -595,6 +595,12 @@
     "author": "elhmn",
     "labels": []
   },
+  {
+    "issue": "https://github.com/osscameroon/miniyotas/issues/26",
+    "title": "Dockerise miniyotas",
+    "author": "elhmn",
+    "labels": []
+  },
```
Comment (elhmn) [inline src/res/osscameroon_issues.json:603]: This file is automatically generated, and should not be modified manually

File: README.md [hunk 2]
```diff
@@ -12,6 +12,19 @@ Yotas are set by one of OSS Cameroon admin on this [spreadsheet](https://docs.go
 ## How to run locally
 Run `yarn dev`
```
Comment (elhmn) [inline README.md:13]: Can you add the `yarn install` step in the documentation, as you mentioned that it was not present in here ?

**Review by elhmn — CHANGES_REQUESTED** [review]
Great work, I just left a comment about the autogenerated json file

File: src/res/osscameroon_issues.json (hunk 1 above)
Comment (andrew21-mch) [reply src/res/osscameroon_issues.json:603]: Alright, so what am i expected to do with the file please, should i remove line 598-603

Comment (elhmn) [reply src/res/osscameroon_issues.json]: Yes those changes should not be part of the PR, so removing those lines would be great

Comment (andrew21-mch) [reply src/res/osscameroon_issues.json]: alright

**Comment by andrew21-mch** [comment]
I just removed the lines of code, is this what is expected ?

**Review by elhmn — APPROVED** [review]

**Pull request merged**